src/tls.py

```diff
@@ -456,11 +460,11 @@ def import_cert(self, alias: str, filename: str) -> None:
             logger.error(e.output)
             raise e
 
-    def remove_cert(self, alias: str) -> None:
+    def remove_cert(self, alias: str, filename: str) -> None:
         """Remove a cert from the truststore."""
         try:
             subprocess.check_output(
-                f"keytool -delete -v -alias {alias} -keystore truststore.jks -storepass {self.truststore_password} -noprompt",
+                f"keytool -delete -v -alias {alias} -keystore truststore.jks -storepass {self.truststore_password} -noprompt ; rm {filename}",
```

Comment:
**todo:** Here I would just pass the alias in the function args. 

Then, in a separate `subprocess.check_output`, call:

```python
f"rm -f {alias}*.pem"
``` 

The separate subprocess gives you more information where things failed. 
The `-f` ensures that you don't get an error for a missing file. 
The `{alias}*.pem` means that you don't need to split out the `{alias}_ca/cert.pem` on L205.

Reply:
I thought about this, I was going to keep the _ca _cert endings as it makes things slightly easier to handle, but It's not a big deal. Is it better to just have the alias as it is unique enough @marcoppenheimer ?